middleware/cache/manager_test.go

```diff
@@ -0,0 +1,23 @@
+package cache
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gofiber/utils/v2"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_manager_get(t *testing.T) {
```

Comment:
@brunodmartins pls add t.Parallel() in parent and child tests

Reply:
Done =)